cfme/tests/cli/test_appliance_console.py

```diff
@@ -1278,9 +1278,8 @@ def test_appliance_console_extend_storage_negative():
     pass
 
 
-@pytest.mark.manual
 @pytest.mark.tier(1)
-def test_appliance_console_static_dns():
+def test_appliance_console_static_dns(temp_appliance_preconfig_long):
```

Comment:
Thoughts on parametrizing this test case against IPV4 vs IPV6 reconfiguration? Then you can move the part that fetches the secondary DNS to a fixture, since that is part of setup.

Reply:
Done